src/scss/pages/_home.scss

```diff
@@ -0,0 +1,89 @@
+.nav-items {
+  padding: 0;
+  float: right;
+  display: block;
+}
+
+.nav-item {
+  display: inline-block;
+  position: relative;
+
+  .nav-link {
+    padding: 20px;
+    color: #ffffff;
+    text-decoration: none;
+    display: block;
+  }
+
+  &:not(.dropdown):before {
```

Comment:
:not(_class) lắm quá.
nghiên cứu thử cho a cái này: 
.nav-items {
  > .nav-item {
  }
}